tutorials/best_practices/scene_organization.rst

```diff
@@ -176,10 +176,13 @@ in another context without any extra changes to its API.
 
     .. code-tab:: csharp
 
+      using Godot;
+      using System;
```

Comment:
These usings look a bit out of place because this code block is actually 3 different files and the _Parent_ part is only an excerpt of a file.